<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>混合方式UI设计</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>771</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actNew"/>
    <addaction name="actOpen"/>
    <addaction name="separator"/>
    <addaction name="actClose"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="actCut"/>
    <addaction name="actCopy"/>
    <addaction name="actPaste"/>
    <addaction name="separator"/>
    <addaction name="actClear"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>格式</string>
    </property>
    <addaction name="actFontBold"/>
    <addaction name="actFontItalic"/>
    <addaction name="actFontUnderLine"/>
    <addaction name="separator"/>
    <addaction name="actToolbarLab"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actNew"/>
   <addaction name="actOpen"/>
   <addaction name="actClear"/>
   <addaction name="separator"/>
   <addaction name="actCut"/>
   <addaction name="actCopy"/>
   <addaction name="actPaste"/>
   <addaction name="separator"/>
   <addaction name="actFontBold"/>
   <addaction name="actFontItalic"/>
   <addaction name="actFontUnderLine"/>
  </widget>
  <action name="actCut">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/cut.bmp</normaloff>:/images/cut.bmp</iconset>
   </property>
   <property name="text">
    <string>剪切</string>
   </property>
   <property name="toolTip">
    <string>剪切</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actCopy">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/copy.bmp</normaloff>:/images/copy.bmp</iconset>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
   <property name="toolTip">
    <string>复制</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actPaste">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/paste.bmp</normaloff>:/images/paste.bmp</iconset>
   </property>
   <property name="text">
    <string>粘贴</string>
   </property>
   <property name="toolTip">
    <string>粘贴</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actFontBold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/bld.bmp</normaloff>:/images/bld.bmp</iconset>
   </property>
   <property name="text">
    <string>粗体</string>
   </property>
   <property name="toolTip">
    <string>粗体</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actFontItalic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/itl.bmp</normaloff>:/images/itl.bmp</iconset>
   </property>
   <property name="text">
    <string>斜体</string>
   </property>
   <property name="toolTip">
    <string>斜体</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actFontUnderLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/under.bmp</normaloff>:/images/under.bmp</iconset>
   </property>
   <property name="text">
    <string>下划线</string>
   </property>
   <property name="toolTip">
    <string>下划线</string>
   </property>
  </action>
  <action name="actClose">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/exit.bmp</normaloff>:/images/exit.bmp</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
  <action name="actOpen">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/open.bmp</normaloff>:/images/open.bmp</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="toolTip">
    <string>打开</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actClear">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/clear.bmp</normaloff>:/images/clear.bmp</iconset>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
   <property name="toolTip">
    <string>清空</string>
   </property>
  </action>
  <action name="actToolbarLab">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>显示工具栏文本标签</string>
   </property>
   <property name="toolTip">
    <string>显示工具栏文本标签</string>
   </property>
  </action>
  <action name="actNew">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/new.bmp</normaloff>:/images/new.bmp</iconset>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
   <property name="toolTip">
    <string>新建</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actCut</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>cut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actClear</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actPaste</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actClose</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actCopy</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
